libcxx/test/std/thread/thread.mutex/thread.lock/thread.lock.guard/mutex.pass.cpp

```diff
@@ -18,31 +18,44 @@
 //     -> lock_guard<_Mutex>;  // C++17
 
 #include <mutex>
```

Comment:
I would suggest that we formulate the tests in terms of the `TestMutex` only, that way they would all become enabled when threading is disabled.

And then we can have a single test (probably in this test file) where we test basic interoperation between `std::lock_guard` and `std::mutex`, and that one would be the only one with `UNSUPPORTED: no-threads`.

Reply:
Done.